LondonBritishLibrary/orient/BLorient680.xml

```diff
@@ -0,0 +1,506 @@
+                                <decoNote xml:id="b1">In the original boards, covered with stamped leather.</decoNote>
+                                <decoNote xml:id="b2" type="bindingMaterial">
+                                    <material key="wood"/>
+                                    <material key="leather"/>
+                                </decoNote>
+                            </binding>
+                        </bindingDesc>
+                        
+                    </physDesc>
+
+                    <history>
+                        <origin>
+                            <origDate notBefore="1711" notAfter="1712" evidence="internal-date"/>
+                        </origin>
+                        <provenance>Looted by the British <persName ref="PRS7484Napier"
+                                >Napier</persName> expedition in <date>1868</date> from the church
+                            of <placeName ref="INS0101MadhaneAlam"/>. </provenance>
+                    </history>
+
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:Wright1877BM"/>
+                                            <citedRange unit="page">66a-67a</citedRange>
+                                        </bibl>
+                                    </listBibl>
+                                    <listBibl type="secondary">
+                                        <bibl><ptr target="bm:Bausi2000Osservazioni"/><citedRange unit="page">84; 93</citedRange></bibl>
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+
+        </fileDesc>
+        <encodingDesc>
+            <projectDesc>
+                <p>Encoded according to TEI P5 Guidelines.</p>
+            </projectDesc>
+            <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+                href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+                <xi:fallback>
+                    <p>Definitions of prefixes used.</p>
+                </xi:fallback>
+            </xi:include>
+        </encodingDesc>
+        <profileDesc>
+            <textClass>
+                <keywords>
+                    <term key="Translation"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Apocrypha"/>
+                    <term key="Poetry"/>
+                </keywords>
+            </textClass>
+
+            <langUsage>
+                <language ident="gez">Gǝʿǝz</language>
```

Comment:
I am nor sure, but maybe English should stay, see line 7, where it is defined as a default language.